### DISCUSSION TITLE
How do I change the outline colour on a text input?

### DISCUSSION THREAD

**Discussion by Chris230291** [twbs/bootstrap#36693] [Q&A] [answered]
With buttons I can set `btn-outline-primary`. However if I set this on a text input it fills in blue when pressed. Is there an equivalent for text inputs?

Thanks,
Chris

EDIT: Here is what I mean...

![image](https://user-images.githubusercontent.com/5328818/177660521-48361cb7-894b-45e0-a6a6-d90476a7dc86.png)

**Comment by mdo** [comment, marked as answer]
If you want a blue border, use the border utilities :).

**Reply by Chris230291** [reply]
Thank you I used `class="form-control border-primary"`
